pkg/ansible/controller/reconcile.go

```diff
@@ -235,6 +241,11 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 		}
 		return reconcileResult, errmark
 	}
+
+	// re-trigger reconcile because of failures
+	if !runSuccessful {
```

Comment:
I don't think this will requeue if we are managing status, I think if we just set the `reconcileResult.Requeue` to true up at https://github.com/operator-framework/operator-sdk/pull/2375/files#diff-9649888b3df1961dad649345545de4c0R221 then it should requeue in all cases, unless it's specifically important that we return an error, in which case we'll just need to make sure we do that in the managed status case as well

https://godoc.org/sigs.k8s.io/controller-runtime/pkg/reconcile#Result

Reply:
Solved 👍 Good spot tks.